thunderbird-todoist/scripts/api_utils.js

```diff
@@ -45,11 +45,24 @@ function getAllProjects() {
   });
 }
 
-function addTask(content, due, projectid, messageContent) {
+function getAllCollaborators(projectid) {
+  let endpoint = "projects/" + projectid + "/collaborators"
```

Comment:
This could be `const`.

This is also missing a semicolon at the end, maybe you forgot to run prettier?